source/ui/IDEAS/IDEASXASScanConfigurationView.cpp

```diff
@@ -398,3 +418,8 @@ void IDEASXASScanConfigurationView::onFluorescenceChoiceChanged(int id)
 {
 	configuration_->setFluorescenceDetector(id);
 }
+
+void IDEASXASScanConfigurationView::onDetectorSettingsChange(const QString &args)
```

Comment:
I'm not sure why it's not showing the rest of the code, which is.

void IDEASXASScanConfigurationView::onDetectorSettingsChange(const QString &args)
{
	detectorSettings_->setText(args);
}